fenix/app/build.gradle

```diff
@@ -250,7 +250,7 @@ android.applicationVariants.all { variant ->
 
     println("----------------------------------------------")
     println("Variant name:      " + variant.name)
-    println("Application ID:    " + [variant.mergedFlavor.applicationId, variant.buildType.applicationIdSuffix].findAll().join())
+    println("Application ID:    " + [variant.applicationId, variant.buildType.applicationIdSuffix].findAll().join())
```

Comment:
Nice! This deprecation used to bother me

Although I don't think this change introduced this "issue" but I see this printed in the build log
`Application ID:    org.mozilla.fenix.debug.fenix.debug`

❓ should we just print the `variant.applicationId` ?

Reply:
I noticed the same, but when upgrading Gradle I prefer not to change existing behavior. And the same value is currently printed in the build log. We could open another issue to investigate if this is intentional or if should we change it.

Reply:
Sounds good! 👍